.golangci.yml

```diff
@@ -240,14 +240,12 @@ linters:
   enable:
     - gofmt
     # - govet
-    # - errcheck
+    - errcheck
     # - staticcheck
     # - unused
     # - gosimple
     # - structcheck
-    - varcheck
```

Comment:
Why is this one deleted?

Reply:
When running, there is a warning that the linter is abandonned by its upstream. The link is also broken on the list, returning a 404, so I do not think we can really rely on that one: https://golangci-lint.run/usage/linters/